tests/utils.py

```diff
@@ -46,3 +46,11 @@ def check_lambda_fixtures(lambda_functions):
     lambda_names = {lam["FunctionName"]: lam for lam in resp["Functions"]}
     for lam in lambda_functions:
         assert lam["FunctionName"] in lambda_names
+
+
+def check_sns_fixtures(sns_topics):
+    client = boto3.client("sns")
+    resp = utils.call(utils.backoff_check, func=lambda: client.list_topics())
+    topic_names = [tpc["TopicArn"].rsplit(":", 1)[-1] for tpc in resp["Topics"]]
+    assert len(topic_names) == len(sns_topics)
+    assert set(topic_names) == set(sns_topics)
```

Comment:
I'd avoid assuming name will be in ARN. I don't think that's guaranteed?
No need to cast to set, since we can assume boto3 and mocked cloud work correctly and only return each item once.

```python
assert len(resp["Topics"]) == len(sns_topics)
```

Alternatively, perhaps generate list of names by calling call get_topic_attributes on each topic arn and getting DisplayName from the response. Then assert set == set

Reply:
"DisplayName" is a different field from "Name" for SNS: [DisplayName is the name for the topic if it publishes user notifications](https://docs.aws.amazon.com/sns/latest/api/API_SetTopicAttributes.html).

Unfortunately, SNS doesn't have an API call available to get that original "Name" given a Topic ARN. I think it would be safe to assume that SNS Topics follow the [ARN spec](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) by having that "Name" field in the arn?

Reply:
Yeah, looks like it. That works for me. I might recommend removing line 55 still, since checking length isn't too important. The fixture should have error-ed out already if you requested two topics with the same name.

Reply:
Good with me. Removed